docs/API.md

```diff
@@ -76,6 +76,7 @@ Jest uses Jasmine 2 by default. An introduction to Jasmine 2 can be found
   - [`testRunner` [string]](#testrunner-string)
   - [`unmockedModulePathPatterns` [array<string>]](#unmockedmodulepathpatterns-array-string)
   - [`verbose` [boolean]](#verbose-boolean)
+  - [`growl` [boolean]](#growl-boolean)
```

Comment:
can we put this and the description below into its correct alphabetical location?